python/pyspark/pandas/groupby.py

```diff
@@ -1171,9 +1173,9 @@ def apply(self, func: Callable, *args: Any, **kwargs: Any) -> Union[DataFrame, S
 
         if is_series_groupby:
             name = psdf.columns[-1]
-            pandas_apply = SelectionMixin._builtin_table.get(func, func)
```

Comment:
@HyukjinKwon Do you remember what this was for?

Reply:
No .. the change seems fine

Reply:
Maybe it's just for better performance by switching to NumPy's?

Reply:
I see. Let's keep it as is for now.